keyboards/decent/numpad/keymaps/default/config.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#define BOOTMAGIC_ROW 1
+#define BOOTMAGIC_COLUMN 0
+#define RGB_MATRIX_SLEEP
+
+#define RGB_MATRIX_LED_COUNT 22
+#define RGB_MATRIX_CENTER {37, 42}
+
+#define JOYSTICK_AXIS_COUNT 4
+#define JOYSTICK_BUTTON_COUNT 10
+#define JOYSTICK_HAS_HAT
```

Comment:
If joysticks are meant to be installed, the configuration for them should live at the keyboard level.

Reply:
You can have a config.h at the keyboard level. The JSON side of things is available but not documented (and may change in future; I want to make some improvements to how this feature is configured).
https://github.com/qmk/qmk_firmware/blob/master/data/schemas/keyboard.jsonschema#L347-L376

Reply:
Makes sense, I'll move it. Looking forward to the joystick config, it'll be nicer to have everything in one place.